percy/lib/percy_options.py

```diff
@@ -1,15 +1,15 @@
 class PercyOptions:
     IGNORE_ERRORS = 'ignoreErrors'
     ENABLED = 'enabled'
-    PERCY_OPTIONS = ['percy:options']
+    PERCY_OPTIONS = ['percy:options', 'percyOptions']
 
     def __init__(self, capabilities):
         self._capabilities = capabilities
         self.percy_options = self._parse_percy_options() or {}
 
     def _parse_percy_options(self):
         options = list(map(self._capabilities.get, self.PERCY_OPTIONS))
-        options = (options[0] if options else {}) or {}
+        options = (options[0] or options[1] if options else {}) or {}
```

Comment:
```suggestion
        options = ((options[0] or options[1]) if options else {}) or {}
```
Just to be safe

Reply:
when we do that it returns us none. Weird i just checked using complier. else is not seems to be working.
same is happening when i am removing `percyOptions`. (old code).